tests/Feature/TrafficAnalytics/StartSentinelTrafficTest.php

```
@@ -0,0 +1,62 @@
+it('passes custom traffic settings as sentinel env', function () {
+    $server = Server::factory()->create(['team_id' => $this->team->id]);
+    $server->settings->is_traffic_analytics_enabled = true;
+    $server->settings->traffic_topn = 100;
+    $server->settings->traffic_sample_threshold = 500;
+    $server->settings->traffic_retention_1h_days = 14;
+    $server->settings->traffic_retention_1d_days = 180;
+    $server->settings->is_geoip_enabled = false;
+    $server->settings->geoip_refresh_days = 7;
+    $server->settings->save();
+
+    $env = StartSentinel::sentinelTrafficEnvironment($server->fresh());
```

PHPStan (2.2.7)

[warning] Parameter `#1` $server of static method App\Actions\Server\StartSentinel::sentinelTrafficEnvironment() expects App\Models\Server, Illuminate\Database\Eloquent\Collection<int, Illuminate\Database\Eloquent\Model>|Illuminate\Database\Eloquent\Model|null given.


(argument.type)